llvm/tools/opt/opt.cpp

```diff
@@ -149,32 +149,30 @@ static cl::opt<bool>
     StripNamedMetadata("strip-named-metadata",
                        cl::desc("Strip module-level named metadata"));
 
-
-
 static cl::opt<bool>
     OptLevelO0("O0", cl::desc("Optimization level 0. Similar to clang -O0. "
-                              "Use -passes='default<O0>' for the new PM"));
+                              "Same as -passes='default<O0>'"));
```

Comment:
@aeubanks: Me and @karka228 is testing the PullRequest funcitonality a bit with this simple adjustment of the options descriptions. So I tried out adding you as an additional reviewer.

Given the rules we got for mixing -O and -passes this new wording will highlight that -O<n> is equivalent to the longer form using -passes. So I think this is a nice improvement.